src/platform/site-wide/cta-widget/index.js

```diff
@@ -461,9 +460,14 @@ export class CallToActionWidget extends React.Component {
       : '';
     const target = isInternalLink ? '_self' : '_blank';
 
+    const buttonText = [
+      this._serviceDescription[0].toUpperCase(),
+      this._serviceDescription.slice(1),
+    ].join('');
+
     return (
       <a className={buttonClass} href={this._toolUrl} target={target}>
-        {titleCase(this._serviceDescription)}
+        {buttonText}
```

Comment:
Removed the title-casing to instead capitalize only the first letter -

![image](https://user-images.githubusercontent.com/1915775/86385928-0b8e1f80-bc5f-11ea-82d7-f39b2f3f421e.png)

https://dsva.slack.com/archives/C52CL1PKQ/p1593705251422300?thread_ts=1593635440.391800&cid=C52CL1PKQ